cli/command/container/opts.go

```diff
@@ -812,14 +842,23 @@ func validDeviceMode(mode string) bool {
 }
 
 // validateDevice validates a path for devices
+func validateDevice(val string, serverOS string) (string, error) {
+	switch serverOS {
+	case "linux":
+		return validateLinuxPath(val, validDeviceMode)
+	case "windows":
+		// Windows does validation entirely server-side
```

Comment:
👍 We should remove more of these validations from the client, and defer as much as possible to the daemon/api. Doing so is much more maintainable, and makes sure that the _daemon_ is in charge as to "what's valid" and "what not" (which may differ between daemon versions). I'll start working on an epic for that